sphinxcontrib/confluencebuilder/__init__.py

```diff
@@ -211,6 +211,8 @@ def setup(app):
     app.add_config_value('confluence_adv_trace_data', None, '')
     # Do not cap sections to a maximum of six (6) levels.
     app.add_config_value('confluence_adv_writer_no_section_cap', None, 'env')
+    # Disable Appending "rest/api" to api calls
```

Comment:
This new configuration is place in the wrong section. It has been added under "configuration - undocumented", but instead should be under "configuration - advanced publishing".

The recently added configuration should also be lexicographically sorted based off the other configuration entries in its section.

(optional) In the comment, "appending" can be lower case; "API can be uppercase; and a period is missing.